packages/cyberstorm/src/components/AntiResult/AntiResult.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import styles from "./AntiResult.module.css";
+import { classnames } from "../../utils/utils";
+
+export interface Props {
+  children?: ReactNode | ReactNode[];
+  className?: string;
+}
+
+/**
+ * Ease of use component set for adding helpful content in
+ * place where content is missing
+ */
+export function AntiResult(props: Props) {
```

Comment:
"anti" refers to being against or opposed to something. I don't think were against showing results. And "results" might make sense if we're always showing search results, but why limit the use of component to that? How about "NoContent" or "EmptyContent"?

Reply:
"NoContent" and "EmptyContent" would imply there is no content, when there's for example the silly looking ghost.

The name of the component is supposed to be kind of the same as "anti-hero".

Reply:
As per the Discord discussion the name is bad and you should feel bad.